doc/nrf/protocols/multiprotocol/index.rst

```diff
@@ -46,3 +46,59 @@ Given the nature of the dynamic multiprotocol solution, take into account the fo
   The application logic must take this into account by providing a callback for all crucial transmissions and handle the ``MAC_TRANSACTION_OVERFLOW (== 241)`` status code.
 * The most time-critical part of the Zigbee stack operation is the Base Device Behavior (BDB) commissioning process.
   Postpone time-consuming operations until the ``ZB_BDB_SIGNAL_DEVICE_FIRST_START`` or ``ZB_BDB_SIGNAL_DEVICE_REBOOT`` signal is received.
+
+.. _multiprotocol_bt_thread:
```

Comment:
Can we refer to this page in the Matter bridge application documentation? We could use it when describing the number of supported Bluetooth LE devices.